language/move-vm/runtime/src/runtime.rs

```diff
@@ -313,6 +337,18 @@ impl VMRuntime {
             log_context,
         )?;
 
+        let mut return_layouts = vec![];
```

Comment:
Might be cleaner to use `return_tys.iter().map(|ty| /* convert to layout */).collect()?`